shared/domains/chelonia/GIMessage.js

```diff
@@ -474,7 +474,7 @@ export class GIMessage {
 
   // $FlowFixMe[unsupported-syntax]
   static [serdesSerializeSymbol] (m: GIMessage) {
-    return [m.serialize(), m.direction(), m.decryptedValue(), m.innerSigningKey()]
```

Comment:
Nice catch! (How did this not get caught before? Shouldn't this have thrown? 🤔 )

Reply:
It should have. This wasn't really used before the SW though, as there was no need for serialisation / deserialisation when not crossing boundaries.